ArduCopter/mode.h

```diff
@@ -1398,8 +1403,8 @@ class ModeZigZag : public Mode {
     bool allows_arming(bool from_gcs) const override { return true; }
     bool is_autopilot() const override { return true; }
 
-    // save current position as A (dest_num = 0) or B (dest_num = 1).  If both A and B have been saved move to the one specified
-    void save_or_move_to_destination(uint8_t dest_num);
+    // save current position as A (dest_num = position::A) or B (dest_num = position::B).  If both A and B have been saved move to the one specified
+    void save_or_move_to_destination(position dest_num);
```

Comment:
This is no longer `dest_num`.

Also, `Destination destination` might be better as we use `Position` for locations-relative-to-origin.